samples/stock/demo/rangeselector-datagrouping/demo.details

```diff
@@ -0,0 +1,10 @@
+---
+name: Rangeselector Datagrouping
+authors:
+  - Torstein Hønsi
+js_wrap: b
+tags:
+  - Highcharts Stock demo
+categories:
+  - General
```

Comment:
I think we should put this in the `Various features` category, otherwise it will be placed under "Chart types".